exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -111,18 +113,20 @@ impl CommandExtension for Finalize {
 
         let service_config = ServiceConfig { module_name, port };
 
-        let jvm_config: JvmConfig = context
+        // Retrieve `EjbConfig` saved at the `GenerateNodeConfig` step
+        let ejb_config: EjbConfig = context
             .get(keys::SERVICES_SECRET_CONFIGS)
             .expect("Can't get services secret configs")
             .get(EJB_JVM_CONFIG_NAME)
```

Comment:
Is it ejb config as the type suggests, or ejb jvm config as the key here suggests?

Reply:
It's ejb config, the name of the key is kinda "outdated"...